packages/lambdas/src/fitbit-subscriber-verification.ts

```diff
@@ -35,8 +35,8 @@ export const handler = Sentry.AWSLambda.wrapHandler(async (event: APIGatewayEven
     }
   }
 
-  capture.message("Fitbit subscriber verification fail", {
-    extra: { context: "webhook.fitbit.subscriber-verification" },
+  capture.message("FitbitSubVerifLambda subscriber verification fail", {
```

Comment:
nit: we already have the context indicating the lambda, so like the auth one we could leave `Fitbit` here for ease of read?